### PR TITLE
added issue templates

### LINKED ISSUE #251: Add issue templates like hpc-stack has...
Since this will become the site where end users go to request new packages, we need to have the issue templates from hpc-stack...

### PR DESCRIPTION
**Pull request by edwardhartnett**
Fixes #251 

Adding issue templates from hpc-stack.

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
Looks good.

File: .github/ISSUE_TEMPLATE/add_package_request.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+name: Add Package Request
+about: Request that a package be added to hpc-stack
```
Comment (kgerheiser) [inline .github/ISSUE_TEMPLATE/add_package_request.md]: Replace hpc-stack with spack-stack (or Spack)

File: .github/ISSUE_TEMPLATE/add_package_request.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+title: ''
+labels: 'New Package'
+assignees: 'Hang-Lei-NOAA'
+
+---
+
+**Please describe the package or library you would like to add to hpc-stack.**
```
Comment (kgerheiser) [inline .github/ISSUE_TEMPLATE/add_package_request.md]: Same here

File: .github/ISSUE_TEMPLATE/add_package_request.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+A description of package including GitHub repo.
+
+**What applications at NOAA will be using this package or library?**
+A list of groups or applications which will use this pacakge or library.
+
+**Is there already a package or library in hpc-stack that provides this, or related, functionality?**
+List any other related pacakges already in hpc-stack, as far as you know.
```
Comment (kgerheiser) [inline .github/ISSUE_TEMPLATE/add_package_request.md]: And here

File: .github/ISSUE_TEMPLATE/add_package_request.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
+---
+name: Add Package Request
+about: Request that a package be added to spack-stack
+title: ''
+labels: 'New Package'
+assignees: 'Hang-Lei-NOAA'
```
Comment (climbfuji) [inline .github/ISSUE_TEMPLATE/add_package_request.md]: I think this needs updating.

File: .github/ISSUE_TEMPLATE/add_package_request.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+---
+
+**Please describe the package or library you would like to add to spack-stack.**
+A description of package including GitHub repo.
+
+**What applications at NOAA will be using this package or library?**
```
Comment (climbfuji) [inline .github/ISSUE_TEMPLATE/add_package_request.md]: Remove "at NOAA" or expand as appropriate, please

File: .github/ISSUE_TEMPLATE/add_package_request.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+A list of groups or applications which will use this pacakge or library.
+
+**Is there already a package or library in spack-stack that provides this, or related, functionality?**
+List any other related pacakges already in spack-stack, as far as you know.
+
+**Additional context**
+Add any other context or screenshots about the package or library here.
+
+**Will This Package be Needed in an Operational Application?**
```
Comment (climbfuji) [inline .github/ISSUE_TEMPLATE/add_package_request.md]: NOAA-specific, but ok to keep I think

File: .github/ISSUE_TEMPLATE/install_request.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+name: Install request
+about: Request an installation of package in the stack on HPC's
+title: '[INSTALL] <title>'
+labels: 'install'
+assignees: 'Hang-Lei-NOAA, kgerheiser, jkbk2004'
```
Comment (climbfuji) [inline .github/ISSUE_TEMPLATE/install_request.md]: Needs updating

**Review by climbfuji — CHANGES_REQUESTED** [review]

**Comment by climbfuji** [comment]
FYI, you can add "[skip ci]" to your commit messages to skip running CI for these PRs that don't touch any code (it will then only build the documentation).

**Comment by kgerheiser** [comment]
@climbfuji does that also allow merging without the required checks?

**Comment by climbfuji** [comment]
> @climbfuji does that also allow merging without the required checks?

I believe it does ... we'll find out.

**Comment by kgerheiser** [comment]
The check will still be required. There's no 'good' workaround, what Github suggests is to create a dummy workflow.

https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks

**Comment by climbfuji** [comment]
> The check will still be required. There's no 'good' workaround, what Github suggests is to create a dummy workflow.
> 
> https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks

Annoying. Well, we may keep the options on for administrators to override the required status checks. We can then decide whether an author's decision to not run CI is justified and merge w/o the status checks to pass. Just another option instead of creating complicated dummy workflows.

File: .github/ISSUE_TEMPLATE/add_package_request.md (hunk 5 above)
Comment (edwardhartnett) [reply .github/ISSUE_TEMPLATE/add_package_request.md]: OK

File: .github/ISSUE_TEMPLATE/add_package_request.md (hunk 6 above)
Comment (edwardhartnett) [reply .github/ISSUE_TEMPLATE/add_package_request.md]: I have specified that this is for NOAA only...

File: .github/ISSUE_TEMPLATE/install_request.md (hunk 7 above)
Comment (edwardhartnett) [reply .github/ISSUE_TEMPLATE/install_request.md]: You want your name there too?

Comment (climbfuji) [reply .github/ISSUE_TEMPLATE/install_request.md]: Would be good, yes. Once I had a chance to talk to my colleague (on leave this week), I'll add him as well.

Comment (edwardhartnett) [reply .github/ISSUE_TEMPLATE/install_request.md]: Who is JONG KIM jkbk2004?

Should he be listed here?

Comment (climbfuji) [reply .github/ISSUE_TEMPLATE/install_request.md]: Jong Kim is from the EPIC team, he is doing the installs of hpc-stack on Cheyenne as far as I know.

**Review by climbfuji — APPROVED** [review]
Thanks for addressing my comments.

**Review by kgerheiser — APPROVED** [review]

**Pull request merged**